### PR TITLE
chore: update ipfs-repo

### PR DESCRIPTION
**Pull request by hugomrdias**
Needs this PR linked https://github.com/ipfs/js-ipfs/pull/1335

### CODE REVIEW

**Comment by VictorBjelkholm** [comment]
@vasco-santos as a lead maintainer, yes. Added write permissions for you to this repository now.

**Comment by diasdavid** [comment]
@vasco-santos weird, you should have had write permissions all the time given that you are in the JS team

image

File: package.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
     "hapi": "^16.6.2",
     "hat": "0.0.3",
     "ipfs-api": "^20.0.1",
-    "ipfs-repo": "^0.19.0",
+    "ipfs-repo": "~0.21.0",
```
[the remark this answers is not in the document]
Comment (diasdavid) [reply package.json]: This PR is blocked until https://github.com/ipfs/js-ipfs/pull/1335 is done given https://github.com/ipfs/js-ipfs-repo/issues/165

**Comment by dryajov** [comment]
@vasco-santos @hugomrdias @VictorBjelkholm I believe that we need to pin the `ipfs-repo` version to `0.20.0`, since anything above that will break due to the changes in which repo errors are being handled in newer versions.

**Comment by vasco-santos** [comment]
I think that we need to release the new versions of `ipfsd-ctl` and `ipfs` (with the CLI errors fixing branch), in order to get everything green again. With `ipfs-repo@0.20.0`, I believe that the @hugomrdias 's CLI errors branch will not work.

**Comment by dryajov** [comment]
> I think that we need to release the new versions of ipfsd-ctl and ipfs (with the CLI errors fixing branch), in order to get everything green again.

Sure, but `ipfsd-ctl` is most likely broken for everybody right now due to the use of `^` instead of `~` for `ipfs-repo` version.

**Pull request closed without merging**